### PR TITLE
added last set

### PR DESCRIPTION
**Pull request by Abdullahjavednesar**

### CODE REVIEW

**Comment by arihantparsoya** [comment]
I think I am going to make changes to code. Let me commit my changes and pass the tests. Then I will merge this

**Comment by arihantparsoya** [comment]
Can you merge the changes. Remove test suits since they are not in sympy format.

**Comment by arihantparsoya** [comment]
remove test_sets

**Comment by arihantparsoya** [comment]
remove `test_rubi_test_suit.py` file.

File: sympy/integrals/rubi/utility_function.py [hunk 1]
```diff
@@ -474,14 +474,6 @@ def SinCosQ(f):
 def SinhCoshQ(f):
     return MemberQ([sinh, cosh, sech, csch], Head(f))
 
-def Rt(val, n):
```
Comment (parsoyaarihant) [inline sympy/integrals/rubi/utility_function.py]: why have you removed this?

**Pull request merged**